src/Tizen.NUI/src/public/Images/ImageLoading.cs

```diff
@@ -499,8 +497,7 @@ public static Size2D GetClosestImageSize(string filename)
         /// </summary>
```

Comment:
"consider rotation" -> "taking the image rotation into account" OR ", rotated according to image metadata".

Reply:
thank you for review.
it's done